static/app/views/issueDetails/streamline/note.tsx

```diff
@@ -129,33 +132,45 @@ function StreamlinedNoteInput({
         : errorJSON.detail?.message || t('Unable to post comment'))) ||
     null;
   return (
-    <NoteInputForm data-test-id="note-input-form" noValidate onSubmit={handleSubmit}>
-      <MentionsInput
-        aria-errormessage={errorMessage ? errorId : undefined}
-        style={mentionStyle({theme, minHeight: 14, streamlined: true})}
-        placeholder={placeholder}
-        onChange={handleChange}
-        onKeyDown={handleKeyDown}
-        value={value}
-        required
+    <Flex column gap={space(0.75)} align="end">
+      <NoteInputForm data-test-id="note-input-form" noValidate onSubmit={handleSubmit}>
+        <MentionsInput
+          aria-label={t('Add a comment')}
+          aria-errormessage={errorMessage ? errorId : undefined}
+          style={mentionStyle({theme, minHeight: 14, streamlined: true})}
+          placeholder={placeholder}
+          onChange={handleChange}
+          onKeyDown={handleKeyDown}
+          value={value}
+          required
+        >
+          <Mention
+            trigger="@"
+            data={suggestMembers}
+            onAdd={handleAddMember}
+            displayTransform={(_id, display) => `@${display}`}
+            markup="**[sentry.strip:member]__display__**"
+            appendSpaceOnAdd
+          />
+          <Mention
+            trigger="#"
+            data={suggestTeams}
+            onAdd={handleAddTeam}
+            markup="**[sentry.strip:team]__display__**"
+            appendSpaceOnAdd
+          />
+        </MentionsInput>
+      </NoteInputForm>
+      <Button
+        priority="primary"
+        size="xs"
+        disabled={!canSubmit}
+        onClick={submitForm}
+        aria-label={t('Submit comment')}
       >
-        <Mention
-          trigger="@"
-          data={suggestMembers}
-          onAdd={handleAddMember}
-          displayTransform={(_id, display) => `@${display}`}
-          markup="**[sentry.strip:member]__display__**"
-          appendSpaceOnAdd
-        />
-        <Mention
-          trigger="#"
-          data={suggestTeams}
-          onAdd={handleAddTeam}
-          markup="**[sentry.strip:team]__display__**"
-          appendSpaceOnAdd
-        />
-      </MentionsInput>
-    </NoteInputForm>
+        {t('Comment')}
+      </Button>
```

Comment:
since we have a button we should probably do the more aria friendly thing and put it inside the form with type=submit

Reply:
this also should remove the need for the onClick since react will handle that via the form